packages/node/src/lib/process-request.ts

```diff
@@ -129,20 +132,20 @@ export default function processRequest(
   req: ExtendedIncomingMessage,
   requestBody?: Record<string, unknown> | string,
   options?: LogOptions
-): Entry['request'] {
-  const protocol = fixHeader(req.headers['x-forwarded-proto'])?.toLowerCase() || getProto(req);
-  const host = fixHeader(req.headers['x-forwarded-host']) || req.headers.host;
+): Request {
+  const protocol = fixHeader(req.headers['x-forwarded-proto'] || '')?.toLowerCase() || getProto(req);
+  const host = fixHeader(req.headers['x-forwarded-host'] || '') || req.headers.host;
 
   const denylist = options?.denylist || options?.blacklist;
   const allowlist = options?.allowlist || options?.whitelist;
 
-  let mimeType: string = null;
+  let mimeType = '';
   try {
-    mimeType = contentType.parse(req.headers['content-type']).type;
+    mimeType = contentType.parse(req.headers['content-type'] || '').type;
   } catch (e) {} // eslint-disable-line no-empty
 
   let reqBody = typeof requestBody === 'string' ? parseRequestBody(requestBody, mimeType) : requestBody;
-  let postData: Entry['request']['postData'] = null;
+  let postData: PostData = null as unknown as PostData;
```

Comment:
this is weird, why is this necessary? I would expect either PostData (the type) to allow null as a possible value, or postData (the variable) to allow null like this:

`let postData: PostData | null = null;`